packages/fastclick/README.md

```diff
@@ -2,6 +2,10 @@
 [Source code of released version](https://github.com/meteor/meteor/tree/master/packages/fastclick) | [Source code of development version](https://github.com/meteor/meteor/tree/devel/packages/fastclick)
 ***
 
+```
```

Comment:
This shouldn't be a code-fence (` ``` `) (as you can see in the [preview](https://github.com/elie222/meteor/blob/f0a2c8cfe4a252cb928b27bc607767d3952eb00e/packages/fastclick/README.md) it breaks the display.  How about removing both fencing lines and just add `>` in front of the paragraph?

Either way, I think a bold `Warning:` in front of it would be even more helpful:

```
> **Warning:** As of late...
```